### DISCUSSION TITLE
Allow Only one Tile-Feature per type

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#21536] [Dashboard] [answered]
Would it be possible to allow only one "Feature" per type?
So that you can't setup the same feature multiple times in a card?

![grafik](https://github.com/user-attachments/assets/78aec902-3301-4978-89dd-0632e22ac733)

**Comment by karwosts** [comment]
I think there could at least be plausible reasons to allow duplicates, given that they are individually configurable.

E,.g. if you have 10 presets, you could add two preset rows, and put 5 icons in the top row and 5 in the bottom row. 

Or maybe a number could want to show both a box and a slider.

**Comment by mvdwetering** [comment, marked as answer]
Having the possibility to have multiple the same tile features is useful (or even critical) for [Custom Features for Tile Cards and More
](https://github.com/Nerwyn/service-call-tile-feature) since each feature can be configured differently.

For example these 4 "Custom Feature Row" tilecard features.
![image](https://github.com/user-attachments/assets/c89ae875-c11d-4e6c-813b-d354bd1ecc1f)

Result in:
![image](https://github.com/user-attachments/assets/299e930b-d69a-40bb-8ea5-a2cf5b357920)

**Reply by ChristophCaina** [reply]
All right, 
Topic closed :)
